doc/content/getting-started/migrating-from-v2/configure-ttnctl.md

```diff
@@ -4,7 +4,7 @@ weight: 15
 description: Install and configure CLI tools
 ---
 
-You will need to use the latest version of `ttnctl`, the CLI for {{% ttnv2 %}}. Follow the [instructions from The Things Network documentation][1]. An overview is given below:
+You will need to use the latest version of `ttnctl`, the CLI for **{{% ttnv2 %}}**. Follow the [instructions from The Things Network documentation][1]. An overview is given below:
```

Comment:
I dislike this personally. The styling becomes inconsistent. If we make everything bold, it will be chaotic